packages/story-editor/src/components/checklist/checks/firstPageAnimation.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useMemo, useCallback } from '@googleforcreators/react';
+import { Link, THEME_CONSTANTS } from '@googleforcreators/design-system';
+import { trackClick } from '@googleforcreators/tracking';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
+import { DESIGN_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+import { useIsChecklistMounted } from '../popupMountedContext';
+
+export function firstPageAnimation(animations) {
+  return Boolean(animations?.length);
+}
+
+const FirstPageAnimation = () => {
+  const isChecklistMounted = useIsChecklistMounted();
+  const animations = useStory(({ state: { pages } }) => pages[0]?.animations);
+
+  const isRendered = useMemo(
+    () => firstPageAnimation(animations),
+    [animations]
+  );
```

Comment:
```suggestion
  const hasFirstPageAnimations = useStory(({ state: { pages } }) => firstPageAnimation (pages[0]?.animations));

  const isRendered = hasFirstPageAnimations;
```

Not sure why all the checklist checks use `useMemo` so much, but wouldn't this be easier & cause less re-renders?

cc @littlemilkstudio in case you have some context on that

Reply:
@swissspidy - I agree, what you suggested was my first approach, the functionally definitely doesn't seem to need `useMemo` - but I was trying to follow the structure of the other `checks`.

Reply:
@swissspidy I dont believe `useMemo` is necessary in this `check` and possibly a few others - Ill remove it here to avoid extra re-renders 👍

cc @BrittanyIRL @BrookeGraham

Reply:
Yea I'm not sure either, but yea, the suggested change looks good to me. should cause less re-renders and the `firstPageAnimation` function isn't expensive to call either